### ISSUE TITLE
London tech meetups portal

### ISSUE THREAD

**Issue by yeo-yeo** [fac18/student-project-ideas#2]
**Describe the problem you'd like to solve**
There are lots of tech meetups/events going on which are great for FAC students.  It can be a bit overwhelming to keep track of all these, although students are really proactive in finding and documenting them (Alex).  The app could also send some reminders (e.g. via Slack or email)

**Who are the stakeholders?**
FAC students! And could be extended to anyone who wants to go to London events.  Could also add location search to make it wider than London.

**What is currently being done to solve this problem?**
Meetup has a lot of these events, but it could be nice to have an app pared back for just tech, and with a way to see if your specific group (e.g. FAC cohort) are going.  We currently use a Google calendar but it would be nice if we could automate this (are there APIs that could give us some data?), and also design a GUI around it.

**Comment by redahaq** [comment]
I really like this idea, Gillian! The slack reminder would be particularly valuable to me as a user.

Some first thoughts:
- Maybe the events could also be tagged with topics eg. front-end, node etc.
- Map with location? 

> First $200-worth of API calls are free
The new pricing strategy employed by Google for their Maps Platform is a freemium one – all users get to make $200-worth of API calls for free each month. That works out as, for example:

> up to 28,000 free loads of Dynamic Maps; or
> up to 100,000 free loads of Static Maps; or
> up to 40,000 free Directions calls; or
> up to 40,000 free Geolocation calls.
> Importantly, the new pricing system allows unlimited free use of simple Dynamic Maps in Embeds and in Mobile Native apps. So, if you’re just serving users a simple map with a marker, say to show your office location, you’ll continue to pay nothing under the new system. However, if you use the Embed API in Directions, Views or Search mode, you will be eligible for billing.
> 

-

**Comment by yeo-yeo** [comment]
Definitely!  Maybe it could also autogenerate the route from Fonthill Road and ETA!
